app/views/items/new.html.erb

```diff
@@ -1,3 +1,9 @@
-<h1> Edit Visit Information </h1>
-<%= render 'form' %>
-<div><%= link_to "Back", root_path, class:"btn btn-sm btn-outline-danger"%></div>
\ No newline at end of file
+<div class="container border border-secondary rounded-lg mt-3">
+  <h1 class="text-center">
+      New Item
+  </h1>
+</div>
+
+<container>
+  <%= render 'form' %>
+</container>
```

Comment:
What do the `<container>` tags here do? (I'm wondering if they can be deleted or if they're needed for something else that's happening.)